Lib/test/test_shutil.py

```diff
@@ -1829,6 +1891,246 @@ def test_move_dir_caseinsensitive(self):
         finally:
             os.rmdir(dst_dir)
 
+
+class _ZeroCopyFileTest(object):
```

Comment:
Do the cpython unit tests get run on the different platforms you have defined here?